widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -100,11 +127,17 @@ export function Quote(props: QuoteProps) {
       }
 
       return {
-        swapper: { displayName: swap.swapperId, image: swap.swapperLogo },
+        swapper: {
+          displayName: getSwapperDisplayName(swap.swapperId, swappers),
+          image: swap.swapperLogo,
+        },
         from: {
           token: { displayName: swap.from.symbol, image: swap.from.logo },
           chain: {
-            displayName: swap.from.blockchain,
+            displayName: getBlockchainShortNameFor(
```

Comment:
It would be great if you could fix blockchain name in confirm wallet page as well.

![Screenshot from 2023-11-21 13-00-34](https://github.com/rango-exchange/rango-client/assets/99180934/5ae2946d-5642-4d1d-b086-58ba82bc15ca)